var/spack/repos/builtin/packages/py-notebook/package.py

```diff
@@ -24,16 +25,27 @@ class PyNotebook(PythonPackage):
     variant('terminal', default=False, description="Enable terminal functionality")
```

Comment:
I would honestly just get rid of this variant. It isn't present in the `setup.py`, and terminado was never required until recent versions.

Reply:
not sure about this as it might break older things (and presumably people cared enough to add it), it was there till the very beginning.

Anyways: Semi-related side rant: github history does not work over renames so `git log var/spack/repos/builtin/packages/py-notebook/package.py ` only works till 2264e30d99d8b9fbdec8fa69b594e53d8ced15a1